src/main/webapp/script.js

```diff
@@ -317,3 +345,39 @@ function buildInput(type, value, className, name) {
   form.name = name;
   return form;
 }
+
+// Get all location input fields, and add the Google Places autofill.
+function addLocationAutofill() {
+  let locationInputs = document.getElementsByClassName('places-autofill');
```

Comment:
I'm only seeing 'places-autofill' used here, and since you're doing completely different things based on if it's 'inputDestination' or 'inputPOI', why not give them different class names?

Reply:
Ok, I removed the 'places-autofill' class and just used the IDs of the HTML elements instead. I initially used the 'places-autofill' class as there is lots of overlap (both enable location autocomplete and have the same 'is-valid' check), but this is OK because I add more input field-specific code to future CLs.

Done!